UT4MasterServer.Services/Scoped/CloudstorageService.cs

```diff
@@ -21,6 +21,23 @@ public sealed class CloudStorageService
 	private readonly IMongoCollection<CloudFile> cloudStorageCollection;
 	private readonly ILogger<CloudStorageService> logger;
 
+	private static readonly string[] commonSystemFileFilenames = new string[]
+	{
+		// yes, 3 of these are misspelled by epic
+		"UnrealTournamentOnlineSettings.json",
+		"UnrealTournmentMCPAnnouncement.json",
+		"UnrealTournmentMCPGameRulesets.json",
+		"UnrealTournmentMCPStorage.json",
+		"UTMCPPlaylists.json"
```

Comment:
```suggestion
		"UTMCPPlaylists.json",
```

Reply:
meh, the list will never change. don't think there's any point to this.

Reply:
Just like `commonUserFileFilenames`, but you added a trailing comma there. I'm just saying that your new code in this same PR have inconsistent formatting